package.json

```
@@ -25,7 +25,9 @@
   "scripts": {
     "prepare": "npm update --dev vl-ui-util && npm run prepare:util",
     "prepare:util": "npm explore vl-ui-util -- npm run util",
-    "test": "npm run test:unit && npm run test:e2e",
+    "performance:generate": "node ./performance/generate.js --iterations=100",
+    "performance:run": "npm run performance:generate && docker-compose -f docker-compose-performance.yml up --force-recreate --exit-code-from sitespeed-firefox",
+    "test": "npm run test:unit && npm run test:e2e && npm run performance:run",
```

Ging je deze niet prefixen met test?
```suggestion
    "test": "npm run test:unit && npm run test:e2e && npm run test:performance:run",
    "test:performance:run": "npm run test:performance:generate && docker-compose -f docker-compose-performance.yml up --force-recreate --exit-code-from sitespeed-firefox",
     "test:performance:generate": "node ./performance/generate.js --iterations=100",
```